From 725b052a926697dd39a595f6e8bdf047b4eb6680 Mon Sep 17 00:00:00 2001
From: Pawan Gupta <pawan.kumarx.gupta@intel.com>
Date: Tue, 14 Jun 2016 16:04:44 -0700
Subject: [PATCH 422/441] acpi: add header linux/pci.h

Add linux/pci.h to resolve compilation issue after merging
puma6 and puma7 source.

Signed-off-by: Pawan Gupta <pawan.kumarx.gupta@intel.com>
---
 drivers/acpi/sleep.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/acpi/sleep.c
+++ b/drivers/acpi/sleep.c
@@ -17,6 +17,7 @@
 #include <linux/suspend.h>
 #include <linux/reboot.h>
 #include <linux/acpi.h>
+#include <linux/pci.h>
 #include <linux/module.h>
 
 #include <asm/io.h>
